test-data/environments/pre-prod.ts

```diff
@@ -0,0 +1,24 @@
+import {TestData} from "../test-data";
+
+export const preProdEnvironmentData: TestData = {
+    events: {},
+    requirements: {},
+    staff: {
+        allocationsTester1: {
+            firstName: "Carlo",
+            lastName: "Veo",
+            staffName: "Veo, Carlo (NPS - PO)",
+        },
+        allocationsTester2: {
+            firstName: "Mark",
+            lastName: "Rees",
+            staffName: "Rees, Mark (NPS - PO)",
```

Comment:
I know these were in the original test but I feel a bit uneasy about having real people in here. Plus if Carlo or Mark change their roles or leave the business or even change their names it will leave us with broken tests.

Reply:
Same here, I've created a ticket to cover sourcing better data: https://dsdmoj.atlassian.net/browse/PI-371

Happy to merge this as is and we'll sort out the data separately?